lib/components/Datepicker/Datepicker.tsx

```diff
@@ -75,7 +80,7 @@ export const Datepicker = React.forwardRef<HTMLInputElement, DatepickerProps>(fu
             ref={ref}
             name={name}
             placeholder={placeholder ? `(${placeholder})` : undefined}
```

Comment:
Placeholder style is now different; no more parens and it seems that only month and year (e.g kk/vvvv)

Reply:
As discussed more in the elsewhere, we should let it be `pp.kk.vvvv` for now, so removing only the parens versus previous implementation.

Reply:
Removed parens and reverted to pp.kk.vvvv in the story.